biogeophys/FatesPlantHydraulicsMod.F90

```diff
@@ -1651,7 +1662,13 @@ subroutine ConstrainRecruitNumber(csite,ccohort, bc_in)
           nmin = min(n, nmin) 
        endif
     end do
-    ccohort%n = min (ccohort%n, nmin) 
+    ! Junyan added the constrian of temperature on recuitment number
+    if (bc_in%t_veg_pa(1) > 273.15_r8) then
```

Comment:
This will use the vegetation temperature from patch 1 for all plants. Another alternative is to use the site-level soil temperature (which is indexed by layer).  See here:

https://github.com/NGEET/fates/blob/master/main/FatesInterfaceTypesMod.F90#L431

Reply:
There are a few options here going forward, we can 1) continue to use the patch vegetation temperature, but pass in the correct patch index, 2) use the soon to be implemented running average feature and use an EMA type vegetation temperature or 3) use the soil temperature (instantaneous or time averaged)
Will create an issue